javaslang/src/main/java/javaslang/collection/Iterator.java

```diff
@@ -1392,24 +1392,37 @@ public T getNext() {
             final Iterator<T> that = this;
             return new AbstractIterator<T>() {
 
-                private T next = null;
+                private T next;
```

Comment:
> Feel free to benchmark it :)

Generally yes :) But I think I need to be the pragmatic developer in this case. Otherwise we will never see 2.1.0. My plan is to resolve 68 issues (tendency increasing because of new tickets). Every day an issue => two months.
Every 3 days an issue => half a year...

But yes, quality is very important!!